src/telliot_feeds/integrations/diva_protocol/contract.py

```diff
@@ -93,6 +77,8 @@ def __init__(
         account: Optional[ChainedAccount] = None,
     ):
         chain_id = node.chain_id
+        # chain id is hard-coded because same on all chains, todo: update telliot-core to remove this
```

Comment:
Not sure what you mean here, did you mean to remove one of the chain_id variables?

Reply:
Removed this comment and setting the chain id used for fetching ABI. Importing ABI from file directly instead.